editor/src/clj/editor/validation.clj

```diff
@@ -46,6 +46,10 @@
   (when (> (id-counts id) 1)
     (format "'%s' is in use by another instance" id)))
 
+(defn prop-contains-url-characters? [id name]
+  (when (re-find #"[#:]" id)
+    (format "%s cannot contain special URL symbols such as '#' or ':'" name)))
```

Comment:
`cannot` -> `should not`, because they definitely can, it just makes it harder to use as url.